apps/avatax/next.config.js

```diff
@@ -31,4 +31,11 @@ const configWithSentry = withSentryConfig(
   },
 );
 
-module.exports = isSentryPropertiesInEnvironment ? configWithSentry : nextConfig;
+const config = isSentryPropertiesInEnvironment ? configWithSentry : nextConfig;
+
+const withBundleAnalyzer = require("@next/bundle-analyzer")({
+  // eslint-disable-next-line turbo/no-undeclared-env-vars
+  enabled: process.env.ANALYZE_BUNDLE === "true",
+});
```

Comment:
If this is always imported and only selectively activated via `enabled` property, why do we need dynamic `require`?

Reply:
copy-pasted from docs, I can change to import ;)

Reply:
done